circleci/dapple-deploy

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
move this into a seperate directory or naming scheme that makes it clear that this is Clever internal.